core/src/test/scala/unit/kafka/tools/ConsoleProducerTest.scala

```diff
@@ -123,4 +163,37 @@ class ConsoleProducerTest {
     assertEquals("console-producer",
       producerConfig.getString(ProducerConfig.CLIENT_ID_CONFIG))
   }
+
+  @Test
+  def testBatchSizeOverride(): Unit = {
+    val config = new ConsoleProducer.ProducerConfig(batchSizeOverride)
+    val producerConfig = new ProducerConfig(ConsoleProducer.producerProps(config))
+    assertEquals("456",
+      producerConfig.getString(ProducerConfig.BATCH_SIZE_CONFIG))
+  }
+
+  @Test
+  def testBatchSizeSet(): Unit = {
+    val config = new ConsoleProducer.ProducerConfig(batchSizeSet)
```

Comment:
renamed to `testBatchSizeSetButMaxPartitionMemoryBytesNotSet`
And also the config variable name `batchSizeSet` -> `batchSizeSetButMaxPartitionMemoryBytesNotSet`